src/module/form.js

```diff
@@ -0,0 +1,7 @@
+const Form = () => `<form id="form" class="row-md-3 row-sd-12">
+  <input type="text" class="input-name" placeholder="Your name">
```

Comment:
- [ ] [REQUIRED] Using the [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png) as a guide, I think it would be better to include the header  `Add Your Score` to your form.